commerce/migrations/0052_fill_order_total.py

```diff
@@ -0,0 +1,23 @@
+from commerce.models import Order
+from django.db import migrations
+from django.db.migrations import RunPython
+
+
+def fill_order_total_field(*args, **kwargs):
+
+    orders_without_total = Order.objects.filter(total=0)
+
+    for order in orders_without_total:
+        order.total = order.calculate_total()
```

Comment:
This will cause troubles with history orders as the supplier could be changed in the settings. We should remove the total model field and keep calculating total value on-the-fly only.